use_case_examples/cifar/cifar_brevitas_finetuning/PerrorImpactOnFMNIST.ipynb

```diff
@@ -323,7 +323,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "As the value of `p_error` increases, the inference time in FHE decreases, but the model's performance is affected. For custom ML models, it is recommended to search in simulated FHE mode to find a `p_error` that offers an interesting compromise between accuracy and execution time. In Concrete ML, we have proposed a binary search algorthm showed in [Searching for the best error probability](advanced_features.md)."
```

Comment:
Why not keep a link to local file?

Reply:
because I'd rather use-case examples (which are outside of docs) link to the official online doc. 

I'd turn your question around: advanced examples link to the github doc.. (not the official gitbook online one), which I find less than ideal

Reply:
ah yes my bad, better for use case examples to link to online doc indeed

however for `advanced examples`, why is it less than ideal to link to go to local doc ? if a path changes, both will be wrong, but local link has the advantage of not having to be connected to the internet